etna/datasets/internal_datasets.py

```diff
@@ -752,6 +752,35 @@ def get_ihepc_dataset(dataset_dir: Path) -> None:
     df_full.to_csv(dataset_dir / f"IHEPC_T_full.csv.gz", index=True, compression="gzip", float_format="%.8f")
 
 
+def get_australian_wine_sales_daataset(dataset_dir: Path) -> None:
+    """
+    Download and save Australian total wine sales by wine makers in bottles.
+
+    This dataset consists of wine sales by Australian wine makers between Jan 1980 – Aug 1994.
+
+    References
+    ----------
+    .. [1] https://www.rdocumentation.org/packages/forecast/versions/8.1/topics/wineind
+    """
+    url = (
+        "https://raw.githubusercontent.com/etna-team/etna/9417d61976305ea5980e91cd06d6f33c6c7c4560/"
+        "examples/data/monthly-australian-wine-sales.csv"
+    )
```

Comment:
I use dataset version from our repository

Reply:
How much is it different?

Reply:
I think there is no difference. `pmdarima` uses the same data: [link](https://github.com/alkaline-ml/pmdarima/blob/15919ef6de7778dfb0e74e9b47708f6ea1ec4ee2/pmdarima/datasets/wineind.py#L19)